### ISSUE TITLE
Zoom level not functional.

### ISSUE THREAD

**Issue by sabicalija** [asterics/WebACS#19]
# Zoom level not adjustable

Changing the position of zoom level slide (see screenshot) has no effect on percent value (100%) or actual zoom level of WebACS.

Low zoom level           |  High zoom level
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/13609768/36486090-f70f81d6-171d-11e8-8243-d7c0fb503d8a.png)  |  ![image](https://user-images.githubusercontent.com/13609768/36486114-00bf61ce-171e-11e8-8d7d-658605819078.png)

**Comment by kingcole** [comment]
yes, sorry, I added that slider a long time ago, but never got round to actually implement the functionality - so, actually it is not so much a bug, but a lack of implementation
